test/asTree.test.ts

```diff
@@ -4,13 +4,17 @@ import { richTextFixture } from "./__fixtures__/richText";
 
 import { asTree } from "../src";
 
-// Keep this test first so that we're sure it's the first one to access the fixture
 test("does not mutate the provided rich text field", (t) => {
-	const richTextJSON = JSON.stringify(richTextFixture.en);
+	const richText = richTextFixture.en;
 
-	asTree(richTextFixture.en);
+	// We will run `asTree` on `clonedRichText` and compare the object to the
+	// original value.
+	const originalRichText = JSON.parse(JSON.stringify(richText));
+	const clonedRichText = JSON.parse(JSON.stringify(richText));
 
-	t.deepEqual(JSON.parse(richTextJSON), richTextFixture.en);
+	asTree(clonedRichText);
+
+	t.deepEqual(originalRichText, clonedRichText);
```

Comment:
I'm afraid this doesn't mitigate the fact that the value of the fixture might mutate if another test runs before, that's why I added a comment about it before ^^

What used to happen basically is when you give in a rich text field that says:
```html
<strong>foo <em>bar</em> baz</strong>
```
It gets it right the first time but mutates the field to something that just says:
```
<strong>foo bar baz</strong>
```
And then it stays like so because the spans tree is now "flat"

Reply:
Oh you're totally right! I think we can solve this by making a `createRichTextFixtures` function that returns new deep copies of the JSON files. Then we'll always have fresh fixtures to work with. This is important because Ava runs tests asynchronously by default, and we should be writing tests that are order agnostic whenever possible.

I'll make this change. Thanks for pointing that out. :)

Reply:
`createRichTextFixtures`:
https://github.com/prismicio/prismic-richtext/blob/f2852f13bec4c80f060b35b719028ea73acf459c/test/__testutils__/createRichTextFixtures.ts#L12-L22

The updated test:
https://github.com/prismicio/prismic-richtext/blob/f2852f13bec4c80f060b35b719028ea73acf459c/test/asTree.test.ts#L7-L16

If this still does not properly test for span mutations, let me know and I'll revert to the original test.

Reply:
Awesome!